beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -70,24 +72,25 @@ func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 				if ctx.Err() != nil {
 					return nil, ctx.Err()
 				}
-				start := start + uint64(i)*step
-				step := step * uint64(len(peers))
-				count := mathutil.Min(count, (helpers.StartSlot(finalizedEpoch+1)-start)/step)
+				stp := step * uint64(len(peers))
```

Comment:
what are stp and str? can we use better names?

Reply:
just not to run over the same vars stp is for step cnt is for count ...

Reply:
stp and str are not clear enough. was there a problem with the variable shadowing?

Reply:
Can we change back to the old names ? The 3 letter vars arent clear enough